packages/genesys-spark-components/src/components/beta/gux-avatar/gux-avatar.tsx

```diff
@@ -32,6 +40,30 @@ export class GuxAvatar {
   @Prop()
   accent: GuxAvatarAccent = 'default';
 
+  /**
+   * Shows presence such as away or available
+   */
+  @Prop()
+  presence: GuxAvatarPresence = 'available';
```

Comment:
If I just use the `presence` property on its own would I be correct in saying it does create any visual change to the element. I would have to couple it with `presenceRing`